core/src/main/scala/com/metamx/tranquility/beam/Beam.scala

```diff
@@ -19,22 +19,37 @@
 package com.metamx.tranquility.beam
 
 import com.twitter.util.Future
+import scala.collection.immutable.BitSet
+import scala.collection.mutable
```

Comment:
This is only being used to get `mutable.BitSet` so that can be used to merge bitsets